### ISSUE TITLE
Foreign error in fnc_handleVirtualMass

### ISSUE THREAD

**Issue by jlillis** [acemod/ACE3#5833]
**Arma 3 Version:** `1.78` (stable)
**CBA Version:** `3.4.1` (stable)
**ACE3 Version:** `3.11.0` (stable)

**Mods:**
```
- CBA_A3
- ace
- acex
- task_force_radio
```

**Description:**
The following error appears in .rpt file on mission init after the 1.78 update:
```

23:01:14 Error in expression <l "_loadCoef") then {
_loadCoef = _unit getUnitTrait "loadCoef";
_unit setVariab>
23:01:14   Error position: <getUnitTrait "loadCoef";
_unit setVariab>
23:01:14   Error Foreign error: Unknown enum value: "loadCoef"
23:01:14 File z\ace\addons\movement\functions\fnc_handleVirtualMass.sqf, line 29
```

**Steps to reproduce:**
- Put a unit down in the editor
- Right-click -> Play as the character
- Check your .rpt file

**Where did the issue occur?**
- Dedicated, Self-Hosted Multiplayer, Singleplayer

**Placed Modules:**
- None

**RPT log file:**
https://pastebin.com/Sv4G8e2P

**Comment by Drofseh** [comment]
I was just about to write an issue for this as well.
I've noticed it when I open the Virtual Arsenal.

ARMA 1.78.143717 only ACE 3.11.0 and CBA 3.4.1 loaded.

https://gyazo.com/6f8b80bae77e2000e0ac686bf15ddee7

RPT:
https://gist.github.com/Drofseh/511ce0d423e2eedb4c3a7c71a11af9ca

**Comment by keithleon** [comment]
i see this after the briefing screen once you're in the game. any ideas?

**Comment by PabstMirror** [comment]
It's a valid command (see biki https://community.bistudio.com/wiki/getUnitTrait)
Just seems to be a mistake in 1.78

For now we're not going to change anything, so the error will be there
Hopefully bis can hotfix

(I normally play with script errors on as well, so I know this is annoying because it sorta blocks the showing of real errors)

**Comment by gmangnall** [comment]
I have opened the ace_movement.pbo file in PBO Manager, copied the virtualmass.sqf file out - edited the file to essentially be blank - deleted the virtualmass.sqf file form the PBO and copied the blank one back in. Error now gone. Of course I might have stopped some functionality form working - but none detected yet and the error is now gone.

**Comment by Drofseh** [comment]
> Error now gone.

The errors we just warnings, nothing was actually broken.
Deleting the function definitely will cause you problems with anything that calls it though.

**Comment by PabstMirror** [comment]
```
["unit", {(_this select 0) setVariable ["ace_movement_loadCoef", 1]}, true] call CBA_fnc_addPlayerEventHandler;
```
Add this in pre-init to a mission or make it a mod and the error will go away

**Comment by PabstMirror** [comment]
Fixed in current 1.80 profiling build (see https://community.bistudio.com/wiki/Arma_3_Steam_Branches)
